clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -225,14 +224,23 @@ class GCEUtil {
           String projectName, String region, List<String> forwardingRuleNames, Compute compute, Task task, String phase) {
     task.updateStatus phase, "Looking up network load balancers $forwardingRuleNames..."
 
-    def foundForwardingRules = compute.forwardingRules().list(projectName, region).execute().items.findAll {
+    def forwardingRules = new SafeRetry<List<ForwardingRule>>().doRetry(
+      { return compute.forwardingRules().list(projectName, region).execute().items },
+      "list",
+      "regional forwarding rules",
+      task,
+      phase,
+      RETRY_ERROR_CODES,
+      []
+    )
+    def foundForwardingRuleNames = forwardingRules.findAll {
```

Comment:
This filter returns the actual forwarding rules, not just the names. The variable name is a bit misleading.